app-admin/rsyslog/rsyslog-8.19.0.ebuild

```diff
@@ -11,15 +11,9 @@ HOMEPAGE="http://www.rsyslog.com/"
 BRANCH="8-stable"
 
 if [[ ${PV} == "9999" ]]; then
-	EGIT_REPO_URI="
-		git://github.com/rsyslog/${PN}.git
-		https://github.com/rsyslog/${PN}.git
-	"
+	EGIT_REPO_URI="https://github.com/rsyslog/${PN}.git"
```

Comment:
Strictly pesaking, you could have kept `git://` as a fallback but it doesn't really matter.